### DISCUSSION TITLE
Api

### DISCUSSION THREAD

**Discussion by dawidmachon** [swiftwave-org/swiftwave#1096] [Q&A] [answered]
Hello, 
is there any chance for some api support? For example, deploy static page or more things? This would be a huge!

**Comment by tanmoysrt** [comment, marked as answer]
Most of our apis are graphql apis. Documentation - https://graphql.docs.swiftwave.org/
There are few rest apis for authentication purpose - https://github.com/swiftwave-org/swiftwave/blob/develop/docs/rest_api.md
